README.md

```diff
@@ -1,5 +1,7 @@
 # HR-TIL
 
+![til](https://cloud.githubusercontent.com/assets/11463275/26385291/8125cf72-400c-11e7-8779-1fdb802c9d6f.png)
```

Comment:
This is pretty neat. Where does the image come from? If it's external to the project (I can't find it in on the master branch), can we add the image to the project? That would prevent the image tag from breaking if the external image was ever deleted or moved.